src/main/kotlin/org/appjam/smashing/domain/auth/social/OidcTokenValidator.kt

```
@@ -0,0 +1,68 @@
+        )
+
+        val claims = Jwts.parserBuilder()
+            .setSigningKey(publicKey)
+            .build()
+            .parseClaimsJws(idToken)
+            .body
+
+        if (claims.issuer != iss) throw CustomException(ErrorCode.INVALID_ISS)
+
+        claims.subject
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)